docs/using.adoc

```diff
@@ -198,7 +197,9 @@ vagrant ssh
 You are now at a shell prompt inside the Vagrant box. You can now
 execute commands and use the tools provided.
 
-You can use the
+=== Using orchestrators
+
+You can use
 https://github.com/projectatomic/adb-utils/blob/master/README.adoc[sccli]
 to manage the orchestration services inside of ADB. `sccli` makes it
 easy to start and stop orchestration providers like Kubernetes or
```

Comment:
@budhrg do you think we still need examples here for kubectl and oc? how do you recommend proceeding here?